js_modules/dagster-ui/packages/ui-core/src/asset-data/AssetLiveDataProvider.tsx

```diff
@@ -141,9 +140,10 @@ export const AssetLiveDataProvider = ({children}: {children: React.ReactNode}) =
 
     const assetStepKeys = new Set(dataForObservedKeys.flatMap((n) => n.opNames));
 
-    const runInProgressId = uniq(
-      dataForObservedKeys.flatMap((p) => [...p.unstartedRunIds, ...p.inProgressRunIds]),
-    ).sort();
+    const runInProgressId: string[] = [];
+    // const runInProgressId = uniq(
+    //   dataForObservedKeys.flatMap((p) => [...p.unstartedRunIds, ...p.inProgressRunIds]),
+    // ).sort();
```

Comment:
I don't totally see how this is related and it seems like it'll have a significant impact since we won't observe any runs for any live data, not just health data?